cmake/toolchain/llvm/Kconfig

```diff
@@ -17,3 +17,20 @@ config LLVM_USE_LLD
 	  Use LLVM built-in lld linker with llvm/clang.
 
 endchoice
+
+choice LLVM_RTLIB
```

Comment:
no changes requested atm, but I think we should consider if selection of runtime library should be made available in a dedicated `lib/runtime/Kconfig` file here: https://github.com/zephyrproject-rtos/zephyr/tree/main/lib

Much like the fact that we have C standard lib selection located together here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/lib/libc/Kconfig

Even for C standard libs only available to certain toolchains, such as arc mwdt, we have C std lib selection in the common file, ref:
https://github.com/zephyrproject-rtos/zephyr/blob/a18761e29861b709b984eca82dc31efe6214fd62/lib/libc/Kconfig#L94-L97

As runtime lib selection is new in Zephyr build config, the current proposed location is fine, but I would like input from @semihalf-duda-patryk  and others, as to what they believe is a logical location.
Currently I think a location close to C standard lib selection is most logical.

Reply:
Moved `Kconfig` to `lib/runtime`